.github/workflows/test-suite-web-e2e-pr.yml

```
@@ -112,26 +119,15 @@ jobs:
         uses: ./.github/actions/run-e2e-tests
         with:
           project: "web"
-          containers: ${{ matrix.CONTAINERS }}
+          containers: "trezor-user-env-unix bitcoin-regtest"
           test-group: ${{ matrix.TEST_GROUP }}
-          additional-grep: ${{ matrix.SMOKE && '(?=.*)' || '(?=.*@webOnly)' }}
+          additional-grep: "(?=.*@webOnly)"
+          fail-fast: 3
           currents-project-id: "Og0NOQ"
           currents-record-key: ${{ secrets.CURRENTS_RECORD_KEY }}
+          currents-api-key: ${{ secrets.CURRENTS_API_KEY }}
```

actionlint (1.7.4)

input "currents-api-key" is not defined in action "Run Playwright E2E Tests" defined at "./.github/actions/run-e2e-tests". available inputs are "additional-grep", "base-url", "containers", "currents-project-id", "currents-record-key", "currents-tags", "e2e-passphrase", "fail-fast", "project", "test-group", "workspace-dependencies"

(action)